stdlib/src/builtin/type_aliases.mojo

```diff
@@ -56,26 +56,42 @@ struct Origin[is_mutable: Bool]:
         is_mutable.value,
         `>`,
     ]
+    """The underlying MLIR type."""
 
     # ===-------------------------------------------------------------------===#
     # Fields
     # ===-------------------------------------------------------------------===#
 
-    var _mlir_origin: Self.type
+    var value: Self.type
+    """The underlying MLIR value."""
```

Comment:
I know we've used the convention of having a `value` field in other structs that wrap MLIR values, but IMO I think we'd be better moving away from that. (`value` isn't very readable at a glance since its such a generic term.)

My preference is that we leave this field name as `_mlir_origin` for the time being. I'm open to reconsidering that though, perhaps as part of a holistic effort to revist all the MLIR-wrapping types in the stdlib to be more consistent about this in general.

Reply:
I can revert the change for this PR np. It was part of my attempt to do #3833, I don't have any string opinion on the name of the field itself either.